string agvId
string msgType
int64 timestamp
string version

float64 mileage
float64 batterySOH
float64 batterySOC
float64 batteryCurrent
float64 batteryVoltage
float32 vehSpeed    
float32 angle 
float64 lng 
float64 lat 
float64 posX 
float64 posY
int32 steerControlMode

int32 cargoloadingStatus	
string cargoVin	

int32 vehCtrlMode	
int32 autoDriveEnable	

int32 platformState

int32 taskId
int32 subTaskIndex	
int32 taskStatus
int32 taskType	
string targetName	
float64 targetX	
float64 targetY	
int32 pathState
float64 speedLimit

int32 navstate
int32 errcode
int32 stopcode
float64 remain_path

int32 nextTaskPathId